libs/components/indicators/src/lib/modules/wait/wait-adapter.service.ts

```diff
@@ -43,10 +43,16 @@ export class SkyWaitAdapterService implements OnDestroy {
       'position',
       'relative',
     );
+    this.#renderer.setStyle(
+      waitEl.nativeElement.parentElement,
+      'isolation',
+      'isolate',
+    );
```

Comment:
We'd talked about using a class, but that gets complicated b/c it's not clear where that class would be defined. It could go with the `sky-wait` component if view encapsulation were disabled.

For now, adding the additional inline style fixes the issue and seems unlikely to conflict.

Reply:
The style would go in the global stylesheet inside theme. I still think it's appropriate to put it there and use a class.

Reply:
Ok. Added the classes.